cadc-uws-server/src/main/java/ca/nrc/cadc/uws/server/JobPersistenceUtil.java

```diff
@@ -147,6 +147,8 @@ public static void constraintPhaseTransition(ExecutionPhase start, ExecutionPhas
                 return;
             if (ExecutionPhase.ABORTED.equals(end))
                 return;
+            if (ExecutionPhase.PENDING.equals(end))
```

Comment:
I would probably allow EXECUTING -> PENDING below as well. Normally, the first thing a JobRunner does is change to EXECUTING (so you know it got past QUEUED) and started running... aftre that, you don't know if the JobRunner will do some work before changing back to PENDING to redirect to a different JobRunner endpoint.

Reply:
Done